test/extended/deployments/util.go

```diff
@@ -461,6 +461,27 @@ func createFixture(oc *exutil.CLI, fixture string) (string, string, error) {
 	return resource, parts[1], nil
 }
 
+func createDeploymentConfig(oc *exutil.CLI, fixture string) (*deployapi.DeploymentConfig, error) {
```

Comment:
@kargakis there is a way to do it without pooling - https://github.com/kargakis/origin/blob/c3ba2ab160b147bebb938e2471fc260ae710a8bf/test/extended/deployments/deployments.go#L993-L996